docs/source/tutorial/index.rst

```diff
@@ -0,0 +1,28 @@
+Tutorial
+===========
+
+The quickstart example uses a simple dataset and is primarily intended to show a user how to run the ingest pipeline and view basic outputs.
+This tutorial will use a more complex study dataset and step through each ingest stage in much more depth.
+
+In this tutorial we will cover:
+
+1. How to configure a complex study for the extract stage:
+
+    - i.e. multiple messy source data files
+2. How to write a transform function for the transform stage
+3. How to run the ingest pipeline from end to end
+4. How to debug during steps 1 thru 3 by:
+
+    - Running stages individually
+    - Viewing stage outputs
+    - Viewing ingest logs
+
+You can find the complete data and source code for the tutorial: TODO
```

Comment:
I think we should have a complete ingest project for the tutorial somewhere in our library. Maybe in: 

```
.
└── docs
    └── tutorial
        ├── My_Study
        │   └── dataset_ingest_config.yml
        │   └── ...
        └── data
            └── ...
```

Reply:
We kinda have it in the tests directory already.

anyway, not

```
└
└
```